### ISSUE TITLE
Can't cast object to dict or Why fill objects data with None?

### ISSUE THREAD

**Issue by lucascosta** [facebook/facebook-python-ads-sdk#226]
The new `AbstractObject` initialiser fills the instantiating object data with `None` values for each field.

I could not find an advantage for this nor a drawback for not doing it. Furthermore, since the fields are being taken from `self.Field.__dict__.values()`, which also returns the `__doc__` property with a `None` value, the object's `_data` dict gains a `None` key with a `None` value which prevents casting with the `dict()` function, useful for implicit serialization:

```python
> from facebookads.adobjects.campaign import Campaign
> dict(Campaign())
...
KeyError: 'None'
```

**Comment by rituparnamukherjee** [comment]
@lucascosta,

This is same as what #222 solves. Will track them together.

**Comment by lucascosta** [comment]
I see @rituparnamukherjee. I just making it clear that, differently from #222 I don't think we should fix it to better have the empty values correctly, but abdicate the practice altogether. I'm patch the constructor in my repos and I haven't ran into any problems so far.

**Comment by rituparnamukherjee** [comment]
Fixed by last commit

**Comment by lucascosta** [comment]
@rituparnamukherjee, will you be interested in Pull Requests from now or you prefer that we contribute only in the issues and you put the code yourself?

**Comment by rituparnamukherjee** [comment]
PR's yes, its just that we are working on 2.7 version of the SDK and we wanted to release a quick fix of pending 2.6 release and this was faster. 

Having said that, yes please do submit PRs.
